### ISSUE TITLE
A11y_Visual Studio Code Services_Docs Python_Testing_Info and Relationships: Name is not defined for the tables present on the page.

### ISSUE THREAD

**Issue by manaswita-chougule** [microsoft/vscode-docs#6322]
### GitHub Tags
#WCAG1.3.1; #A11ySev3; #Info and Relationships; #A11yTCS; #A11yWCAG2.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_Apr2023; #JAWS; #ScreenReader;
### Actual Result
Name is not defined for the tables on the page. So JAWS is not announcing the name for tables once we hit the "T" key.
Note:
Similar issue is observed for tables throughout the application.

### Expected Result
As the page has multiple tables, unique names should be defined for the tables on the page. So JAWS will announce the name for tables once we hit the "T" key.

### Environment Details
Application Name: Visual Studio Code Services
URL: [https://code.visualstudio.com/](https://code.visualstudio.com/)
Microsoft Windows 11 Enterprise 10.0.22621 Build 22621
Chrome Version 113.0.5672.93 (Official Build) (64-bit)
JAWS : Version 2023

### Repro Steps

1. Hit the application URL: https://code.visualstudio.com/
2. Turn on JAWS.
3. Tab till "Docs" link and press enter.
4. Tab till "Python" link in the left navigation and press enter.
5. Tab till "Testing" link which is present under the python and press enter.
6. Hit the key "T", JAWS will start to navigate through the tables on page.
7. Observe that JAWS is announcing the names for the tables on the page.

### User Impact
Screen reader users will not able to understand the purpose and get context of the tables if unique name is not defined for them.
### Attachment:
JAWS
![Name is not defined for the tables present on the page](https://github.com/microsoft/vscode-docs/assets/91586101/4b673622-8b27-4047-870b-c307803e67a3)

NVDA 
![Name is not defined for the tables present on the page_NVDA](https://github.com/microsoft/vscode-docs/assets/91586101/34eccaed-64e2-4ef6-a09a-815bf31ebee1)

**Comment by rzhao271** [comment]
@gregvanl the issue here is that the tables are missing captions. If there is a Python docs team, could someone from that team look into adding captions to the tables under https://code.visualstudio.com/docs/python/testing?

**Comment by manaswita-chougule** [comment]
#Regressed:05-30-23; #Closed;

Verified the issue in below environment:
URL : https://code.visualstudio.com/docs/python/testing#_general-python-settings
Chrome : Version 113.0.5672.127 (Official Build) (64-bit)

The issue is fixed as Unique column header name(for first column) in table is defined and JAWS is announcing it correctly, so each table can be uniquely identified by screen reader users. Hence closing the bug. Please find below attachment. Thanks!
![TABLE_NAME](https://github.com/microsoft/vscode-docs/assets/91586101/eb167afe-6de3-4b16-b05c-89e16579dd3a)
